libraries/RW/K8s/k8s_connection_mixin.py

```diff
@@ -114,6 +114,36 @@ def shell(
         logger.info("shell stdout: %s", rsp.stdout)
         return rsp.stdout
 
+    def template_workload(
+        self,
+        workload_name: str,
+        workload_namespace: str,
+        workload_container: str
+    ) -> str:
+        """Take in the workload variables and construct a valid string that specifies the namespace and container. 
+
+        Args:
+            workload_name (str): a workload type in which a pod can be found such as deployment/my-deployment or statefulset/my-statefulset
+            workload_namespace (str): a kubernetes namespace or openshift project name
+            workload_container (str): a specific container within a pod, as pods may not default to the desired container
+
+        Returns:
+            workload: a string containing the the expanded workload parameters.
+        """
+        # Check if the namespace is provided in the workload name and return the vlaue verbatim
+        if " -n" or " --namespace" in workload_name: 
```

Comment:
Change
`" -n" or " --namespace" in workload_name`
to
`" -n" in workload_name or " --namespace" in workload_name`
Otherwise it will always evaluate to True since` " -n"` is a truthy value in python

Reply:
thanks for the catch!